portal/config/eproms/Organization.json

```diff
@@ -2717,6 +2717,34 @@
       },
       "resourceType": "Organization"
     },
+    {
+      "extension": [
+        {
+          "timezone": "Europe/Madrid",
+          "url": "http://hl7.org/fhir/StructureDefinition/user-timezone"
+        },
+        {
+          "research_protocols": [
+			{"name": "IRONMAN v3"}
+          ],
+          "url": "http://us.truenth.org/identity-codes/research-protocol"
+        }
+      ],
+      "id": 146117,
+      "identifier": [
+        {
+          "system": "http://pcctc.org/",
+          "use": "secondary",
+          "value": "146-117"
+        }
+      ],
+      "language": "es_ES",
+      "name": "Hospital Universitario 12 de Octubre",
```

Comment:
Wow, wouldn't believe that was a real name if it didn't have a [wiki page](https://es.wikipedia.org/wiki/Hospital_12_de_Octubre) :male_detective: